docs/source/utils.md

```diff
@@ -97,6 +102,41 @@ For example:
 python -m osp.core.pico install city
 ```
 
+### Using pico as a Python module
+
+Pico can also be used within the Python shell. In particular, four functions 
+`install`, `uninstall`, `packages` and `namespaces` are 
+available to be imported from the `osp.core.pico` module.
+
+```python
+from osp.core.pico import install, namespaces, packages, uninstall
+```
+
+Each function is used in a similar way to its command-line counterpart.
+
+- `install`: accepts _one or more_ positional arguments of string
```

Comment:
There is a slight inconsistency with the CLI section about where each function got its own subsection. Would it be perhaps a bit better to make this symmetric?

Reply:
Isn't that too verbose? The CLI question walks you slowly through the functionality of pico, and then the Python module section basically tells you that all this functionality that you are already familiar with (because you've just read it) is also available from the Python shell. I would not see adding a sub-section for each function again as an improvement.

If we do something regarding this, I would rather unify everything, so that there is a section for each kind of functionality (install, uninstall, list), that tells you how to use it both on the CLI and the Python shell. But this can also overwhelm the new users (which are likely to be only interested in the CLI functionality at first) with too much information, so we have to be careful.

Reply:
I see what you're saying. Maybe an easy solution would be to increase the level of subsections of the CLI parts. For example  `### pico installs` would become `#### pico installs`. What do you think?

Reply:
I committed something to try to fix this, have a look again.